web/models.py

```
@@ -2790,3 +2790,116 @@ class ScheduledPost(models.Model):
+        ordering = ['-updated_at']
+
+    def __str__(self):
+        return f"Chat session with {self.user.username} - {self.created_at.strftime('%Y-%m-%d %H:%M')}"
+
+class AIChatMessage(models.Model):
+    """Model for storing individual messages in AI chat sessions."""
+    ROLE_CHOICES = [
+        ('user', 'User'),
+        ('assistant', 'Assistant'),
+        ('system', 'System'),
+    ]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)